code/game/area/Space Station 13 areas.dm

```diff
@@ -1323,7 +1323,7 @@ proc/process_adminbus_teleport_locs()
 
 /area/solar
 	requires_power = 0
-	dynamic_lighting = 0
```

Comment:
See comment on what dynamic_lighting actually does. I have no idea why this was set to 0 in goonlights, normally it would mean solars would be perpetually in the dark. I think this